whelktool/scripts/2020/09/lxl-3253-remove-languageNote.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+ * Remove all occurrences of marc:languageNote
+ *
+ * See LXL-3253 for more information.
+ */
+
+failedIDs = getReportWriter("failed-to-delete-bibIDs")
+scheduledForChange = getReportWriter("scheduledForChange")
+
+selectByCollection('bib') { bib ->
+    Map work = getWork(bib)
+    boolean changed = false
+
+    if (!work) {
+        return
+    }
+
+    if (work.remove('marc:languageNote')) {
```

Comment:
From what I can surmise from discussion in Issue the script does not need to take into account if translationOf is present?

Reply:
Exactly. MSS has analyzed the data based on log files. They have come to the conclusion that it should be ok to remove all occurrences of `marc:languageNote`. In most cases where `marc:languageNote` is missing, it will be better and in some cases the data is incorrectly catalogued. But it will at least be consistent.